Updated notebooks following ECMWF+ENTSO-E reviews

GioAldriICS opened the pull request:
This PR introduces a revised and cleaned-up set of Jupyter Notebooks for the 2025 ECMWF Training repo.

The book has been added a title and introduction, it's been splitted  into parts, notebooks titles have been revised, download notebooks merged, climate-change-assesment notebook heavily modified, all main comments addressed, including some recent suggestions/feedback from ENTSO-E.

Once more, the notebooks (now 8: unused ones have been cleaned up) make use of data from the PECD v4.2 dataset on the CDS and cover various use cases relevant to the energy and climate sectors.

l2ghics committed:
Updated notebooks following ECMWF+ENTSO-E reviews, updated book, removal of unnecessary notebooks

nicrie commented:
thanks @GioAldriICS ! Could you please turn on GH Pages by following [these steps](https://github.com/ecmwf-training/jupyterbook-template?tab=readme-ov-file#13-enable-github-pages)? That'll let us preview your changes as a Jupyter Book and make reviewing much easier.

GioAldriICS commented:
Hi Niclas,




I turned them on. Let me know if it's enough for you to review or if any other action is needed.


Thanks,




Best regards,




Giovanni









From "Niclas Rieger" ***@***.***

To "ecmwf-training/2025-entsoe-training" ***@***.***

Cc "Giovanni Aldrigo" ***@***.***,"Mention" ***@***.***

Date Mon, 15 Sep 2025 07:22:40 -0700

Subject Re: [ecmwf-training/2025-entsoe-training] Updated notebooks following ECMWF+ENTSO-E reviews (PR #3)



nicrie left a comment (ecmwf-training/2025-entsoe-training#3)

thanks @GioAldriICS ! Could you please turn on GH Pages by following these steps? That'll let us preview your changes as a Jupyter Book and make reviewing much easier.


—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>

nicrie commented:
Hi @GioAldriICS , it seems something is missing. In your repository, you should see a workflow (called **build-deploy-book** that should have run successfully, showing a flow diagram on the right hand side. But somehow the flow diagram is not showing up.
<img width="658" height="429" alt="image" src="https://github.com/user-attachments/assets/2267f361-4e17-41af-a2ff-adb7357d9a15" />

-----------------

Just to be sure: did you use the worflow dispatcher on the right hand side of the **Action** panel, the button reading **Run workflow**? It has to be triggered manually once.  You should then get a flow diagram showing a link to the rendered JupyterBook. 

<img width="1304" height="429" alt="image" src="https://github.com/user-attachments/assets/14c673e0-f0b9-4166-b4ba-389d7b46f281" />

GioAldriICS commented:
Hi Niclas,


ok you should see it now. Please, let me know either way.


Best regards,


Giovanni









From "Niclas Rieger" ***@***.***

To "ecmwf-training/2025-entsoe-training" ***@***.***

Cc "Giovanni Aldrigo" ***@***.***,"Mention" ***@***.***

Date Tue, 16 Sep 2025 06:28:20 -0700

Subject Re: [ecmwf-training/2025-entsoe-training] Updated notebooks following ECMWF+ENTSO-E reviews (PR #3)



nicrie left a comment (ecmwf-training/2025-entsoe-training#3)

Hi @GioAldriICS , it seems something is missing. In your repository, you should see a workflow (called build-deploy-book that should have run successfully, showing a flow diagram on the right hand side. But somehow the flow diagram is not showing up.

image.png (view on web)


Just to be sure: did you use the worflow dispatcher on the right hand side of the Action panel, the button reading Run workflow? It has to be triggered manually once. You should then get a flow diagram showing a link to the rendered JupyterBook.

image.png (view on web)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>

nicrie commented:
Thanks @GioAldriICS ! 

For later reference, I put the link to the rendered book here: https://gioaldriics.github.io/2025-entsoe-training/

l2ghics committed:
second round of corrections to code

l2ghics committed:
slight modification to download notebook to avoid error message

l2ghics committed:
updated cross references and book

GioAldriICS commented:
Hi Niclas,
we addressed the issues with the notebooks. An email by Letizia will shortly follow to include Nube. Download output cells were removed, the download notebook was modified to get rid of the error message: this was due to the CDS recently increased cost limit for large request. Letizia will complement in the email. The rest should be addressed. Let me know is something is still missing/off, thank you!

l2ghics committed:
corrections to 2n review by technocal officer
